src/v/cluster/data_migration_backend.cc

```diff
@@ -522,6 +522,10 @@ ss::future<errc> backend::do_topic_work(
   state sought_state,
   const inbound_topic_work_info& itwi,
   tsws_lwptr_t tsws) {
+    vlog(
+      dm_log.trace,
+      "itwi.cloud_storage_location = {}",
```

Comment:
can we add more context here f.e. the topic, the `itwi` is not very informative

Reply:
I made it logged in the universal `do_topic_work` instead, both for in- and outbound migrations